core/app/services/spree/cart/empty.rb

```diff
@@ -0,0 +1,11 @@
+module Spree
+  module Cart
+    class Empty
+      prepend Spree::ServiceModule::Base
+
+      def call(order:)
+        order.empty! ? success(order) : failure(order)
```

Comment:
@KacperMekarski let's move the contents/logic of the `empty!` method here and call this service in the `empty!` method :)

Reply:
done :)